tests/Makefile

```diff
@@ -36,11 +36,11 @@ ifeq ($(DISTRO),RHEL4)
 endif
 
 ifeq ($(DISTRO),RHEL5)
-    SUBDIRS:=$(filter-out bounds inet_socket mmap nnp_nosuid overlay unix_socket, $(SUBDIRS))
+    SUBDIRS:=$(filter-out bounds inet_socket mmap nnp_nosuid overlay unix_socket atsecure mac_admin, $(SUBDIRS))
 endif
 
 ifeq ($(DISTRO),RHEL6)
-    SUBDIRS:=$(filter-out nnp_nosuid overlay, $(SUBDIRS))
+    SUBDIRS:=$(filter-out nnp_nosuid overlay atsecure, $(SUBDIRS))
 endif
```

Comment:
technically, AT_SECURE was supported and enforced all the way back, so an alternative to skipping it out would be to just conditionally include the right header on RHEL5/RHEL6, possibly the kernel auxv header or the elf one.

Reply:
There is no header file on default RHEL 5 / 6 installation that contains the definition of getauxval() function, so this doesn't solve the problem.

Reply:
Ok, that's fine then.